service/src/main/java/tds/config/repositories/impl/TimeLimitConfigurationRepositoryImpl.java

```diff
@@ -49,12 +49,12 @@ public TimeLimitConfigurationRepositoryImpl(NamedParameterJdbcTemplate jdbcTempl
 
         Optional<TimeLimitConfiguration> maybeTimeLimitConfig;
         try {
-            maybeTimeLimitConfig = Optional.of(
+            maybeTimeLimitConfig = Optional.ofNullable(
```

Comment:
This shouldn't make a functional difference, but technically only non-null values should be passed into Optional.of =)